reddit/reddit/settings.py

```diff
@@ -116,6 +118,12 @@
 
 USE_TZ = True
 
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=60),
+    'REFRESH_TOKEN_LIFETIME': timedelta(days=30),
+    'AUTH_HEADER_TYPES': ('Bearer',),
+    'BLACKLIST_AFTER_ROTATION': True
+    }
```

Comment:
The closing brace should not have any indent, it should be aligned with 'S' from the 'SIMPLE_JWT' variable